lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -578,10 +439,79 @@ def _partition_pdf_batched_named(
     def _run_pdfminer(pdf_path, hash_key, use_cache):
```

Comment:
Once this goes in, where does this get called? I guess from the managed-service before we update that?

Reply:
Yes, exactly. I want to be able to update Sycamore in managed-service without breaking it.